client/src/main/proto/app/cash/backfila/service.proto

```diff
@@ -75,3 +75,18 @@ message CreateAndStartBackfillRequest {
 message CreateAndStartBackfillResponse {
   optional uint64 backfill_run_id = 1;
 }
+
+// Checks on the status of a backfill.
+message CheckBackfillStatusRequest {
```

Comment:
`Get` seems like our typical verb here instead of Check

Reply:
I didn't want to have it confused with the dashboard action

Reply:
oh I see

Reply:
> I'm curious what you'll use this for
Gonna use it to automate stock splits. Part of the procedure is to run a backfill in portfolios